modules/tinymce/src/themes/silver/test/ts/webdriver/editor/DisabledNestedMenuItemTest.ts

```diff
@@ -90,4 +103,16 @@ describe('webdriver.tinymce.themes.silver.editor.menubar.DisabledNestedMenuItemT
     await RealKeys.pSendKeysOn(servicesMenuItemSelector, [ RealKeys.combo({}, 'arrowright') ]);
     assertServicesMenuIsOpen();
   });
+
+  it('TINY-7700: Enabled menu item with children should have arrow icon enabled', async () => {
+    await pOpenCodeMenu();
+    const menuItem = UiFinder.findIn(SugarBody.body(), servicesMenuItemSelector).getOrDie();
+    assert.isFalse(Class.has(menuItem, disabledClass));
```

Comment:
This should be using a `UiFinder.exists` with the class in the selector (and the same for the other test below).